src/libraries/System.Private.Xml/src/System/Xml/Schema/NamespaceList.cs

```diff
@@ -363,7 +363,7 @@ public static NamespaceList Intersection(NamespaceList o1, NamespaceList o2, boo
                 }
                 if (!v1Compat)
                 {
-                    if (o1._targetNamespace == string.Empty)
+                    if (o1._targetNamespace.Length == 0)
```

Comment:
How do we know that _targetNamespace isn't null here?

Reply:
```suggestion
                    if (o1._targetNamespace == string.Empty)
```